src/sentry/models/groupinbox.py

```diff
@@ -153,3 +153,14 @@ def get_inbox_details(group_list):
     }
 
     return inbox_stats
+
+
+def get_inbox_reason_text(inbox_reason: GroupInboxReason):
+    if inbox_reason == GroupInboxReason.NEW:
+        return "New issue"
+    elif inbox_reason == GroupInboxReason.REGRESSION:
+        return "Issue regressed"
+    elif inbox_reason == GroupInboxReason.ONGOING:
+        return "Issue ongoing"
```

Comment:
Would "Regressed issue" and "Ongoing issue" work better? ("skinned rabbit" vs "rabbit skinned") I always have trouble in my head to figure out what is the right way.